src/ucs/async/thread.c

```diff
@@ -298,20 +299,39 @@ static void ucs_async_thread_mutex_cleanup(ucs_async_context_t *async)
     }
 }
 
+static inline int poll_events_convert_ucs_event_sets(short poll_event)
+{
+    int event_set_event = 0;
+
+    if (poll_event & POLLIN) {
+        event_set_event |= UCS_EVENT_SET_EVREAD;
+    }
+    if (poll_event & POLLOUT) {
+        event_set_event |= UCS_EVENT_SET_EVWRITE;
+    }
+    if (poll_event & POLLERR) {
+        event_set_event |= UCS_EVENT_SET_EVERR;
+    }
+    return event_set_event;
+}
+
 static ucs_status_t ucs_async_thread_add_event_fd(ucs_async_context_t *async,
                                                   int event_fd, int events)
 {
     ucs_async_thread_t *thread;
     ucs_status_t status;
+    int event_set_event;
 
     status = ucs_async_thread_start(&thread);
     if (status != UCS_OK) {
         goto err;
     }
 
+    event_set_event = poll_events_convert_ucs_event_sets(events);
```

Comment:
`event_set_events`?

Reply:
I think it's a strange name too.
Do you have any idea about a proper name?
I would like to identify, event variable `poll` and `event_set`
* poll -> poll_event;
* event_set -> event_set_event.

What do you think `es_event`?

Reply:
Yes, good idea

* poll -> poll_events
* event_set -> es_events

we have to use the plural to emphasize that it can be `POLLIN|POLLUT` and not just either `POLLOUT` or `POLLIN`

Reply:
fixed